dev/package.json

```diff
@@ -518,13 +518,15 @@
         "@types/socketio-wildcard": "^2.0.1",
         "@types/stack-trace": "0.0.29",
         "chai": "^4.2.0",
+        "fs-extra": "^8.0.0",
```

Comment:
It looks like this one and the corresponding @types are switched, this should be the proper dep and the types should be the devDep